cabal-install/Distribution/Client/Setup.hs

```diff
@@ -416,6 +419,18 @@ filterConfigureFlags flags cabalLibVersion
     -- Cabal < 1.3.10 does not grok the '--constraints' flag.
     flags_1_3_10 = flags_1_10_0 { configConstraints = [] }
 
+configPackageDB' :: ConfigFlags -> PackageDBStack
```

Comment:
While you're at it, would be nice to add Haddock comments here to make it more clear what's the difference between these two and the normal versions.